src/pkg/egress/syslog/syslog_connector.go

```diff
@@ -166,12 +166,4 @@ func (w *SyslogConnector) emitLoggregatorErrorLog(appID, message string) {
 	w.logClient.EmitLog(message, option)
 }
 func (w *SyslogConnector) emitStandardOutErrorLog(appID, scheme, url string, missed int) {
-	errorAppOrAggregate := fmt.Sprintf("for %s's app drain", appID)
```

Comment:
Did you mean to remove the function body here?

Reply:
No, this is an oversight, This one was really hindering in test output clarity.
I will roll this one back.